src/object-type-registery.js

```diff
@@ -53,7 +53,7 @@ class ObjectTypeRegistery {
     const objectTypeFieldsKeys = Object.keys(objectTypeFields)
     const newFields = {}
 
-    const fieldsKeys = Object.keys(fields)
+    const fieldsKeys = fields['_doc'] ? Object.keys(fields['_doc']) : Object.keys(fields)
```

Comment:
ew but oki

Reply:
If you can find another way, I'm all hears haha <3